packages/office-ui-fabric-react/src/components/Shimmer/Shimmer.styles.ts

```diff
@@ -85,6 +76,31 @@ export function getStyles(props: IShimmerStyleProps): IShimmerStyles {
         right: '0'
       }
     ],
+    shimmerGradient: [
+      classNames.shimmerGradient,
+      {
+        position: 'absolute',
+        top: 0,
+        left: 0,
+        width: '100%',
+        height: '100%',
+        background: `${shimmerColor || palette.neutralLighter}
+                      linear-gradient(
+                        to right,
+                        ${shimmerColor || palette.neutralLighter} 0%,
+                        ${shimmerWaveColor || palette.neutralLight} 50%,
+                        ${shimmerColor || palette.neutralLighter} 100%)
+                      0 0 / 90% 100%
+                      no-repeat`,
+        transform: `translate3d(-${BACKGROUND_OFF_SCREEN_POSITION}, 0, 0)`,
+        animationDuration: '1.5s',
```

Comment:
Why `1.5s`?

Reply:
The answer above also answers this question